### DISCUSSION TITLE
Latest textual code_browser.tcss error in stylesheet

### DISCUSSION THREAD

**Discussion by esawdust** [Textualize/textual#4467] [Q&A]
Just installed textual and textual-dev and was starting to run through the examples, but hit a snag right off with the code_browser.py example.  Wondering if this is an issue with the latest or a problem with my environment, but suspect the example just broken as I can run the 
`python -m textual` without any issues.

Environment:
Ubuntu 22.04 Jammy
WSL
Anaconda
Python 3.7.16

Here's the console output when trying to run the code_browser.py example:  

`(captain_env) lcox@DESKTOP-DBFUCUV:~/.../examples (main)$ python code_browser.py ../
 Error in stylesheet:
 /home/lcox/seasats/textual/examples/code_browser.tcss:3:5
╭───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│    1 Screen {                                                                                                                                                                                             │
│    2 │   background: $surface-darken-1;                                                                                                                                                                   │
│ ❱  3 │   &:inline {                                                                                                                                                                                       │
│    4 │   │   height: 50vh;                                                                                                                                                                                │
│    5 │   }                                                                                                                                                                                                │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
 • Expected one of 'comment line', 'comment start', 'declaration name', 'declaration set end', or 'whitespace'.                                                                                               • Did you forget a semicolon at the end of a line?                                                                                                                                                          (captain_env) lcox@DESKTOP-DBFUCUV:~/.../examples (main)$`

**Comment by davep** [comment]
What version of Textual are you using to run this with? Apparently your Python version is 3.7 and Textual dropped support for that a while back, which might suggest you're using an old Textual version to run code from the current repo?
